analyses/tp53_nf1_score/02-qc-rna_expression_score.Rmd

```diff
@@ -128,3 +130,57 @@ negative correlation estimate -0.139 at p value 0.297
 Overall distribution of expression and classifier score shows 
 negligible correlation so we cannot directly use the expression/classifier 
 score to infer functionality/phenotype of TP53 inactivation in samples
+
+### Sample_id matched polya and stranded
+
+```{r}
+
+sample_id_multi_library <- clinical %>%
+  filter(experimental_strategy=="RNA-Seq") %>%
+  group_by(sample_id) %>%
+  summarise(counts = length(unique(RNA_library)),
+            RNA_library = toString(RNA_library)) %>%
+  filter(counts > 1)
+
+sample_id_multi_library 
+
+```
+
+```{r}
+
+sample_id_multi_library_scores_df <- clinical %>%
+  left_join(score_stranded_df , by=c("Kids_First_Biospecimen_ID"="sample_id")) %>%
+  left_join(score_polya_df,by=c("Kids_First_Biospecimen_ID"="sample_id"),
+            suffix = c("_stranded","_polya")) %>%
+  filter(sample_id %in% sample_id_multi_library$sample_id) %>%
+  group_by(sample_id,) %>%
+  # because 7316-85 has multiple stranded I'm taking a mean here 
+  # A tibble: 8 x 2
+  #   Kids_First_Biospecimen_ID tp53_score_stranded
+  #   <chr>                                   <dbl>
+  # 1 BS_59ZJWJTF                            0.299 
+  # 2 BS_QYPHA40N                            0.0665
+  # 3 BS_SB12W1XT                            0.229 
```

Comment:
In my opinion, you are better off _showing this output_ in a chunk rather than including it in a comment that could easily get out of date.

Reply:
thanks for the review! That's a great point, I've now added a chunk to display the output